project.clj

```diff
@@ -123,7 +123,8 @@
                                    [leiningen-core "2.9.10" :exclusions [org.clojure/clojure
                                                                          commons-codec
                                                                          com.google.code.findbugs/jsr305]]
-                                   [cider/piggieback "0.5.3"]]}
+                                   [cider/piggieback "0.5.3"]
+                                   [nubank/matcher-combinators "3.8.8"]]}
```

Comment:
Isn't it a bit of an overkill to introduce a new dep (even if it's just for tests) for a single usage?

Reply:
We were lucky to introduce it, since it revealed a bug in our matcher-combinators integration

And answering to the q, Inspector resutls can be quite intrincate so the alternative was quite verbose.

This library also simplified a bespoke mechanism in Orchard tests (the deleted `t/assert-expr 'match?`) https://github.com/clojure-emacs/orchard/commit/9cb236c875773cd90f79aa78e163512fdd8dabb4